src/main/java/bio/terra/cli/command/app/passthrough/Gcloud.java

```diff
@@ -1,11 +1,44 @@
 package bio.terra.cli.command.app.passthrough;
 
+import bio.terra.cli.businessobject.Context;
+import bio.terra.cli.businessobject.Resource;
+import bio.terra.cli.exception.SystemException;
+import bio.terra.cli.exception.UserActionableException;
+import com.google.common.collect.ImmutableList;
+import java.util.ArrayList;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import picocli.CommandLine;
 import picocli.CommandLine.Command;
 
 /** This class corresponds to the second-level "terra gcloud" command. */
-@Command(name = "gcloud", description = "Call gcloud in the Terra workspace.")
+@Command(
+    name = "gcloud",
+    description = "Call gcloud in the Terra workspace.",
+    modelTransformer = Gcloud.SubCmdFilter.class)
 public class Gcloud extends ToolCommand {
 
+  static class SubCmdFilter implements CommandLine.IModelTransformer {
+    public CommandLine.Model.CommandSpec transform(CommandLine.Model.CommandSpec commandSpec) {
+      try {
+        if (!Context.getServer().getCloudBuildEnabled()) {
+          commandSpec.removeSubcommand("--gcs-bucket");
+        }
+      } catch (SystemException e) {
+        return commandSpec;
+      }
+      return commandSpec;
+    }
+  }
+
+  private static final Logger logger = LoggerFactory.getLogger(Gcloud.class);
+
+  @CommandLine.Option(
+      names = "--gcs-bucket",
+      description =
+          "Resource name (not bucket name) of GCS bucket resource. Required for (and only used for) `gcloud builds submit`.")
```

Comment:
I don't see where you are requiring this flag.

On second thought, let's not require. Then we don't have to change code. :) So this can be

```
"Optional flag only used for `terra gcloud builds submit`. Resource name (not bucket name) of GCS bucket where Cloud Build source and logs will be stored.")
```